### PR TITLE
Loading user information into database

### PR DESCRIPTION
**Pull request by RidwanHamid501**
User, Subscription and Keywords tables loaded into RDS. Addresses issue #70

### CODE REVIEW

File: dashboard/dashboard.py [hunk 1]
```diff
@@ -56,13 +108,13 @@ def check_phone_number(phone_number: str) -> bool:
 
 def check_user(phone_number: str, first_name: str, last_name: str) -> bool:
     """Check if the user exists in the database and verify their name."""
-    query = """SELECT * FROM "user"
+    query = """SELECT user_id FROM "user"
```
Comment (SurinaCS) [inline dashboard/dashboard.py:111]: specifying is good!

File: pipeline/output.csv
Comment (Kurt812) [inline pipeline/output.csv:1]: Let's not push this

File: dashboard/dashboard.py [hunk 2]
```diff
@@ -136,39 +193,67 @@ def user_verification() -> None:
         st.error(st.session_state["verification_error"])
 
 
-def topic_submission() -> None:
-    """UI for submitting a topic in the left panel"""
+def topic_and_subscription_ui() -> None:
+    """UI for topic entry and keyword subscription."""
```
Comment (SurinaCS) [inline dashboard/dashboard.py:197]: very specific nice

**Review by Kurt812 — CHANGES_REQUESTED** [review]

**Review by SurinaCS — APPROVED** [review]
looks good!

File: pipeline/load.py
Comment (Kurt812) [inline pipeline/load.py:1]: Nice development, adapting the code to suit our goal

File: pipeline/api.py [hunk 3]
```diff
@@ -43,5 +30,5 @@ def get_topics():
 
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=5000, debug=True)
-    # app.run(debug=True)
+    # app.run(host="0.0.0.0", port=5000, debug=True)
+    app.run(debug=True)
```
Comment (Kurt812) [inline pipeline/api.py:34]: I guess this means the port doesn't need to be specified here?

**Review by Kurt812 — APPROVED** [review]

**Pull request merged**